js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -99,6 +99,21 @@ export class MiniAppBridge {
       );
     });
   }
+
+  /**
+   * Associating requestCustomPermissions function to MiniAppBridge object
+   * @param {string[]} permissionTypes, Array of type of custom permissions that is requested
+   */
+  requestCustomPermissions(permissionTypes: string[]) {
```

Comment:
Also, I guess we need to have one more parameter for a description (which will be used by the host app while showing the custom permissions request)
`"The developer must specify a reason for each permission request."`
Please check [MINI-952](https://jira.rakuten-it.com/jira/browse/MINI-952)

Reply:
I think, we can work with the "reason" parameter in a separate PR? Is it possible?

Reply:
The `description` will passed within the string array, so I don't think anything needs to be done here.

Reply:
That's right @corycaywood , we actually changed the request  string object after the comment here. So I don't think any change is required here.